examples/kserve_deployment/steps/pytorch_steps/mnist_handler.py

```diff
@@ -19,7 +19,8 @@ class MNISTDigitClassifier(ImageClassifier):
     """
     MNISTDigitClassifier handler class. This handler extends class ImageClassifier from image_classifier.py, a
     default handler. This handler takes an image and returns the number in that image.
-    Here the post-processing method has been overridden while others are reused from parent class."""
+    Here the post-processing method has been overridden while others are reused from parent class.
```

Comment:
Will `darglint` complain about this?

Reply:
I ran `darglint` and `ruff` successfully before locally. Waiting for the checks to run first.